modules/sdk-coin-dot/src/dot.ts

```diff
@@ -49,6 +51,38 @@ export interface VerifiedTransactionParameters {
   prv: string;
 }
 
+interface SweepRecoveryOptions {
+  signatureShares: SignatureShares[];
+}
+
+interface SignatureShares {
+  txRequest: TxRequest;
+  tssVersion: string;
+  ovc: Ovc[];
+}
+
+interface TxRequest {
+  transactions: OvcTransaction[];
+  walletCoin: string;
+}
+
+interface OvcTransaction {
+  unsignedTx: DotTx;
+  signatureShares: any[];
```

Comment:
Same as the sol sweep, are we for some reason unable to make this a type other than `any` ?